### ISSUE TITLE
formatter has no reaction

### ISSUE THREAD

**Issue by colorsu** [microsoft/vscode#117545]
Issue Type: <b>Feature Request</b>

你好，我使用alt+shift+F 不能格式化我的文档，完全没有反应。我已经设置了clang-format或者prettier的格式化工具，都没有效果，右键选中文档进行格式化也没有任何反应。
请帮忙解决，谢谢

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:04.245Z)
OS version: Windows_NT x64 10.0.19042

**Comment by colorsu** [comment]
Hello, I can't format my document using alt+shift+F, there is no response at all. I have set the clang-format or prettier formatting tools, but they have no effect, and there is no response after right-clicking and formatting the document.
Please help to solve it, thank you

**Comment by colorsu** [comment]
Hello, I can't format my document using alt+shift+F, there is no response at all. I have set the clang-format or prettier formatting tools, but they have no effect, and there is no response after right-clicking and formatting the document.
Please help to solve it, thank you

**Comment by rzhao271** [comment]
Hi, does running the 'Format Document' command from the command palette, work? Also, maybe check the `editor.defaultFormatter` setting to see if that has been configured properly.

**Comment by colorsu** [comment]
hello, Running 'Format Document' command doesn't work. I changed the editor.defaultFormatter before, It didn't work also. I change it from clang-format to vsformat and has no reaction. 
And I tried to used the prettier, and run the 'Format Document' command also has no reaction.
I can use the 'Format Document' command earlier, but I have no idea why it doesn't work from when. I also tried to reinstall the VSC, and still didn't work.
I tried many ways to make it work, but failed. So I ask for help here.
Thank you very much for your effort. VSC is great.
